src/pages/TokenDetails/index.tsx

```diff
@@ -160,7 +151,11 @@ export default function TokenDetails() {
             <AddressSection address={tokenQueryData.address ?? ''} />
           </LeftPanel>
           <RightPanel>
-            <Widget defaultToken={widgetToken} onReviewSwapClick={onReviewSwap} />
+            <Widget
+              // A null token is still loading, and should not be overridden.
```

Comment:
i don't think this is well documented but i do think we should make explicit if this is a common pattern of null vs undefined

Reply:
I think it's pretty common with the token codepaths